setup.py

```diff
@@ -60,14 +60,16 @@
         'azdev.utilities',
     ],
     install_requires=[
+        'azure-cli-core',
```

Comment:
Just curious, why this one is required? It is not required in the demo project clidev.

Reply:
Because they use functions in this lib in multiple spots (like getting extensions, helpgen.py, linter __init__.py . This how we fixed those import issues. We could remove the dependencies if we had more time to work on this.